src/ol_infrastructure/infrastructure/applications/dagster/Pulumi.applications.dagster.QA.yaml

```diff
@@ -4,3 +4,11 @@ config:
   aws:region: us-east-1
   dagster:db_password:
     secure: v1:y15TVAw5jqVNTrXB:zN/lS+0N6NeqUxTGAwY64IV5LFyXHzDRZloAZwwgQA==
+  dagster:domain: pipelines-qa.odl.mit.edu
+  dagster:instance_type: t3a.medium
+  saltstack:api_password:
+    secure: v1:jcxzqwIOoxQ2261K:PsHmBDKsQetmclqZPtQmAD43OgsnLlDu9NAVVVP9iNtNzPmFNb1fj7QeBANm1uxyhY1cMN7spWYR2w==
+  saltstack:api_url: https://salt-qa.odl.mit.edu
+  saltstack:api_user: pulumi
+  saltstack:key_name: salt-qa
+  vault:skip_tls_verify: "true"
```

Comment:
Is this missing the `vault:address`?

Reply:
That can be supplied by an environment variable. I need to add some details to the readme for that.